components/x-gift-article/src/GiftArticle.jsx

```diff
@@ -288,7 +305,7 @@ const withGiftFormActions = withActions(
 			userIsAHighlightsRecipient && userHasNotYetSavedSharedAnnotations && highlightsHaveNotBeenRemoved
 
 		const initialState = {
-			title: 'Share this article:',
+			title: 'Share this article With:',
```

Comment:
"With" should be "with", right?

```suggestion
			title: 'Share this article with:',
```